.github/workflows/make-test.yml

```diff
@@ -96,7 +96,10 @@ jobs:
     - name: CMake
       env:
         CXXFLAGS: ${{ matrix.cxxflags }}
-      run: 'cd /build && cmake ${GITHUB_WORKSPACE} -DENABLE_DOXYGEN=OFF'
+        EXTRA_CMAKE_FLAGS: ${{ matrix.extra_cmake_flags }}
+      run: 'cd /build && cmake ${GITHUB_WORKSPACE} -DENABLE_DOXYGEN=OFF ${EXTRA_CMAKE_FLAGS}'
```

Comment:
These lines look to be left over from a previous experiment, since there's no `extra_cmake_flags` in the matrix anymore.

Reply:
though it's probably a handy tool, right, this shouldn't be here.